modules/collectd/manifests/package.pp

```diff
@@ -16,8 +16,7 @@
 class collectd::package (
   $collectd_version = '5.4.0-3ubuntu2',
   $yajl_package = 'libyajl2',
-  $apt_mirror_hostname = 'apt.production.alphagov.co.uk',
-
+  $apt_mirror_hostname = undef,
```

Comment:
Isn't this invalid as a default value?

Reply:
It's valid and consistent with the existing code but it's not ideal as a missing hieradata entry will result in broken URLs rather than an aborted run. We'll get this in to make all the apt_mirror lines the same and make the default better across the code base in a further PR.